docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -7,11 +7,54 @@ sidebar_label: Enhanced Project Workflow
 import Tabs from "@theme/Tabs"
 import TabItem from "@theme/TabItem"
 
-:::caution
-This is a **BETA** feature and, as such, is currently under active development. Pay attention using it.
+:::info
+This is a **BETA** feature and, as such, is currently under active development.
+
+It can be safely used but there may be still be some missing features, check the [roadmap section](#roadmap-and-future-improvements) below to find out more.
 :::
 
-The **Enhanced Project Workflow** feature aims to provide a better developer experience when working with the Console, as well as pave the way for the adoption of previously unavailable opportunities, such as the [GitOps deployment strategy](/development_suite/deploy/gitops-based/index.md).
+The **Enhanced Project Workflow** feature aims at providing a better developer experience when working with Mia-Platform Console, as well as pave the way for the adoption of previously unavailable opportunities, such as the [GitOps deployment strategy](/development_suite/deploy/gitops-based/index.md).
+
+## Benefits of the Enhanced Workflow
+
+The technological changes underlying the new workflow allows for improvements in both *user experience* and *future feature development*, as such there are several benefits of adopting the **Enhanced Project Workflow**:
+
+### Tailored orchestrator configurations
+
+By generating the orchestrator objects (e.g. *Kubernetes Deployment files*) at deploy time, the **generated configuration files can now be tailored for the selected environment**. 
+
+Generated configurations are still saved on your Git provider repository in different directories, based on the deployed environment.  
+You can review the configuration prior the commit by leveraging the [review feature](#kubernetes-configuration-review-and-export).
+
+### GitOps Support
+
+As mentioned one of the most relevant features of the **Enhanced Project Workflow** is the GitOps deployment strategy support.
```

Comment:
```suggestion
As mentioned, one of the most relevant features of the **Enhanced Project Workflow** is the GitOps deployment strategy support.
```

Reply:
included in @albertotessarotto suggestion